### ISSUE TITLE
[feature]: Create a github action for publishing sdk to maven central 

### ISSUE THREAD

**Issue by Sarthak160** [keploy/java-sdk#127]
### Is there an existing feature request for this?

- [X] I have searched the existing issues

### Summary

- Create a github action pipeline for publishing a multimodule project to maven central .
- Currently every release of java-sdk is performed via local environment .
- Build an action which will first run the `mvn clean install` , upadate the version `mvn versions:set -DnewVersion=N.N.N` and then make necessary changes to release the sdk to [maven central](https://search.maven.org/artifact/io.keploy/keploy-sdk) .

### Why should this be worked on?

To automate the releasing process for java-sdk

**Comment by Kshitiz-Mhto** [comment]
hello @Sarthak160 , i am new to this Organization and exploring the codebase. would you assign it to me. i want to work on this.

**Comment by Kshitiz-Mhto** [comment]
@Sarthak160 i am building CI/CD pipeline from scratch , but today i found a 3rd party [java-maven-library-publisher](https://github.com/marketplace/actions/java-maven-library-publisher) which will do the work. 

Do u want me to build from scratch or use this 3rd party.

**Comment by Sarthak160** [comment]
Yes @Kshitiz-Mhto . We can try that if it works for multi module project

**Comment by Kshitiz-Mhto** [comment]
@Sarthak160 should i go for the PR  with gh-action using 3rd third party publisher? then we can test whether it is capable for publishing multi-module project.
